### DISCUSSION TITLE
where to set URL for Password-Reset-Mail in OpenSlides 4?

### DISCUSSION THREAD

**Discussion by weingaertnerit** [OpenSlides/OpenSlides#6346] [Q&A]
Hi, 

i had set up Version 4 as described. When I request a password reset email, it only contains an incomplete link. The protocol and hostname are missing. Where can I put this?

Thanks in advance
Markus

**Comment by ostcar** [comment]
I can confirm that this behaviour. It seems, there is currently no settings for the protocol and hostname. Here is the template, that has no placeholder for this two values: https://github.com/OpenSlides/openslides-backend/blob/26788dd5e465e54267eddb5a1d338931438b05cb/openslides_backend/action/actions/user/forget_password.py#L15-L20

I created an issue to track to progress for this: https://github.com/OpenSlides/openslides-backend/issues/1187
